### DISCUSSION TITLE
Are these notices in the bref desktop app "normal"?

### DISCUSSION THREAD

**Discussion by ctessmer** [brefphp/bref#1370] [Support] [answered]
I'm tailing my logs in the bref desktop app (for macOS), and I get this set of notices on each invocation:

```bash
[03-Jan-2023 01:17:56] NOTICE: [pool default] 'user' directive is ignored when FPM is not running as root
[03-Jan-2023 01:17:56] NOTICE: fpm is running, pid 10
[03-Jan-2023 01:17:56] NOTICE: ready to handle connections
[03-Jan-2023 01:17:56] NOTICE: systemd monitor interval set to 10000ms
```

Regarding the first notice,`user directive`, is this expected? Or do I have something messed up in my configuration? If so, any pointers on how to fix?

If normal, can I hide these?

**Comment by mnapoli** [comment, marked as answer]
Hi, these are "normal" indeed (i.e. there is no problem here). I would like to remove those entirely, not hide them in the desktop app but also not log them at all to CloudWatch (because as you point out they can be confusing), but I haven't found a solution yet 🤷‍♂️

**Reply by ctessmer** [reply]
Ahh thank you!

I've never contributed to a library before, but maybe I can help with documentation as a beginner. Because this would be a nice thing to note in the "Logs" section.

> "Don't worry if you see this set of notices. This is normal & expected behavior."
